src/qibocal/calibrations/niGSC/basics/utils.py

```diff
@@ -108,3 +108,11 @@ def gate_fidelity(eff_depol: float, primitive=False) -> float:
     if primitive:
         infidelity /= 1.875
     return 1 - infidelity
+
+
+def number_to_str(number: complex):
+    if np.iscomplex(number):
+        the_str = "{:.2f}{:.2f}j".format(np.real(number), np.imag(number))
```

Comment:
Not sure how are you going to read them, but for manual inspection this is rather confusing (difficult to split the two numbers)
```suggestion
        the_str = "{:.2f} {:.2f}j".format(np.real(number), np.imag(number))
```

Reply:
Good point, thanks, I have to add the sign between the real and imaginary part.